joomla/joomla-cms
# [4.0] Namespace com_templates

Pull Request for Issue # .

### Summary of Changes
This simple PR converts com_templates component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev with sample data, then apply patch (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_templates.zip )

2. Access to backend, access to Extensions -> Templates, test the manage Templates and Styles, make sure it is working as before

### Notes
By looking at the code, look like there is a function to compile less files. However, the library classes like JLess is not available anymore, If I remember correctly, the classes were removed. So I guess we should remove that option from template manage ?
